locations/spiders/franprix_fr.py

```diff
@@ -0,0 +1,50 @@
+import json
+
+from scrapy import Spider
+from scrapy.http import JsonRequest
+
+from locations.hours import OpeningHours
+from locations.items import GeojsonPointItem
+
+
+class FranprixSpider(Spider):
+    name = "franprix_fr"
+    item_attributes = {
+        "brand": "Franprix",
+        "brand_wikidata": "Q2420096",
+    }
+    start_urls = ("https://www.franprix.fr/xhr-cache/resource/stores",)
+
+    def parse(self, response):
+        data = response.json()
+        store_data = data.get("list")
+
+        for store in store_data:
+            oh = OpeningHours()
+            for day in store.get("hours"):
+                day_str = day.get("day").title()[:2]
+                opening_hours = list(day.get("hours"))
+                if not opening_hours:
+                    continue
+                formatted_hours = opening_hours[0].lstrip("T").split("/")
+                oh.add_range(
+                    day=day_str,
+                    open_time=formatted_hours[0],
+                    close_time=formatted_hours[1],
+                    time_format="%H:%M",
+                )
+            properties = {
+                "ref": store.get("id"),
+                "name": f"Franprix - {store.get('store_name')}"
+                if store.get("store_name")
+                else "Franprix",
+                "street_address": store.get("street"),
```

Comment:
`street` has city and postcode, so I'd have put it in `addr_full`, not `street_address`.